foreign_cc/private/framework.bzl

```
@@ -680,6 +685,10 @@ def _define_outputs(ctx, attrs, lib_name):
 
     out_include_dir = ctx.actions.declare_directory(lib_name + "/" + attrs.out_include_dir)
 
+    out_data_dirs = []
+    for dir in attr_out_data_dirs:
+        out_data_dirs.append(ctx.actions.declare_directory(lib_name + "/" + dir))
```

Maybe just for sanity's sake?
```suggestion
        out_data_dirs.append(ctx.actions.declare_directory(lib_name + "/" + dir.lstrip("/")))
```

Not sure if that syntax works and not sure if this is actually a problem? Maybe double check but it'd be nice to either reject malformed input here or correct for it.